grpcutil/dns_resolver.go

```diff
@@ -101,8 +101,8 @@ func parseTarget(target string) (host, port string, err error) {
 	return "", "", fmt.Errorf("invalid target address %v", target)
 }
 
-// Resolve creates a watcher that watches the name resolution of the target.
-func (r *dnsResolver) Resolve(target string) (Watcher, error) {
+// Resolve creates a watcher that watches the SRV record resolution of the target.
```

Comment:
It watches both. SRV record first, and if it fails, then it does hostname resolution. If we will skip SRV lookup for service == "", we should document it here.

Reply:
@pstibrany Thanks, improved the comment.